engine/dlib/src/dmsdk/external/remotery/Remotery.h

```diff
@@ -109,6 +109,11 @@ documented just below this comment.
 #define RMT_D3D11_RESYNC_ON_DISJOINT 1
 #endif
 
+// If RMT_USE_INTERNAL_HASH_FUNCTION is defined, the internal function is used implemented.
+// If set to 0, it adds an extern reference to this rmt_HashString32(const char* s, int len, rmtU32 seed)
+#ifndef RMT_USE_INTERNAL_HASH_FUNCTION
+#define RMT_USE_INTERNAL_HASH_FUNCTION 1
+#endif
```

Comment:
I've added this PR: https://github.com/Celtoys/Remotery/pull/216
to hopefully get full support for this addition to the Remotery api.